langchain/agents/agent.py

```diff
@@ -203,24 +220,34 @@ def _call(self, inputs: Dict[str, str]) -> Dict[str, Any]:
             # If the tool chosen is the finishing tool, then we end and return.
             if isinstance(output, AgentFinish):
                 if self.verbose:
-                    langchain.logger.log_agent_end(output, color="green")
+                    self._get_callback_manager().on_tool_start(
```

Comment:
@agola11 note this - i am calling the tool start/end even when its finishing

this is mostly for getting stdout tracer to work nicely

alternative would be an explicit "end agent" endpoint

thoughts?

Reply:
cool ill add